### DISCUSSION TITLE
Support for PNG in Personal info

### DISCUSSION THREAD

**Discussion by LibraryLibrarian** [saicaca/hexo-theme-vivia#74] [General]
When setting up user-related services using vivia, I want to set up services that are not supported by fontawesome, but it seems like there is no provided method to do so using png icons or similar.
Is it possible to set up icons for any third-party service using other means?

**Comment by saicaca** [comment]
This cannot be achieved by editing the config for now; you may need to modify the `profile.ejs` to add your own SVGs.

**Reply by LibraryLibrarian** [reply]
Thank you for your reply.
I understand.

P.S.
I've learned that you, Saicaca, are creating a new blog template called Fuwari, so I'm thinking of switching to it.

**Reply by saicaca** [reply]
Thanks for your support. Switching to Fuwari is a solution to this since it supports all [Iconify icons](https://iconify.design/), not just FontAwesome icons.
Fuwari can actually be considered the next major version of Vivia, and I am going to discontinue the developments of Vivia.

**Reply by LibraryLibrarian** [reply]
Is that so! 
Actually, I realized that Fuwari become an Astro framework following Vivia's design, and I was wondering if Vivia would continue to be supported in the future, so I was thinking of asking separately. 
Thank you for the valuable information 😄
